src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp

```diff
@@ -1595,58 +1595,129 @@ namespace TerminalAppLocalTests
     {
         {
             std::vector<winrt::hstring> args{ L"wt.exe" };
-            VERIFY_ARE_EQUAL(WindowingBehaviorUseNew,
-                             appImpl::AppLogic::_doFindTargetWindow({ args }, WindowingMode::UseNew));
+            auto result = appImpl::AppLogic::_doFindTargetWindow({ args }, WindowingMode::UseNew);

```

Comment:
It might be helpful to a future debugger to leave either a general comment or a `Log` comment per block here as to the general gist of each test section.